src/ground_truth_nsrts.py

```diff
@@ -1963,6 +1965,167 @@ def _get_touch_point_gt_nsrts() -> Set[NSRT]:
     return nsrts
 
 
+def _get_stick_point_gt_nsrts() -> Set[NSRT]:
+    """Create ground truth NSRTs for StickPointEnv."""
+    robot_type, point_type, stick_type = _get_types_by_names(
+        CFG.env, ["robot", "point", "stick"])
+    Touched, InContactRobotPoint, InContactRobotStick, InContactStickPoint, \
+        Grasped, HandEmpty, NoPointInContact = _get_predicates_by_names(
+            CFG.env, ["Touched", "InContactRobotPoint", "InContactRobotStick",
+            "InContactStickPoint", "Grasped", "HandEmpty", "NoPointInContact"])
+    RobotTouchPoint, PickStick, StickTouchPoint = _get_options_by_names(
+        CFG.env, ["RobotTouchPoint", "PickStick", "StickTouchPoint"])
+
+    nsrts = set()
+
+    # RobotTouchPointFromNothing
+    robot = Variable("?robot", robot_type)
+    point = Variable("?point", point_type)
+    parameters = [robot, point]
+    option_vars = [robot, point]
+    option = RobotTouchPoint
+    preconditions = {
+        LiftedAtom(HandEmpty, [robot]),
+        LiftedAtom(NoPointInContact, []),
+    }
+    add_effects = {
+        LiftedAtom(Touched, [point]),
+        LiftedAtom(InContactRobotPoint, [robot, point])
+    }
+    delete_effects = {LiftedAtom(NoPointInContact, [])}
```

Comment:
i really wonder if we should try side predicate learning with this lol. it's worth a shot right? it's just the nsrt learning command with `--strips_learner backchaining`